content/en/docs/Reference/catalog-templates.md

```diff
@@ -97,21 +93,6 @@ properties:
   value:
     packageName: example-operator
     version: 0.1.0
-- type: olm.bundle.object
```

Comment:
Should we put an `olm.csv.metadata` property here in place of the old `olm.bundle.object` properties?

Reply:
Nevermind. I now see "skipping large bundle metadata objects" in the sentence above.